src/content/reference/react/useReducer.md

```diff
@@ -34,31 +34,31 @@ function MyComponent() {
   // ...
 ```
 
-[See more examples below.](#usage)
+[Aşağıda daha fazla örneğe bakın.](#usage)
 
-#### Parameters {/*parameters*/}
+#### Parametereler {/*parameters*/}
 
-* `reducer`: The reducer function that specifies how the state gets updated. It must be pure, should take the state and action as arguments, and should return the next state. State and action can be of any types. 
-* `initialArg`: The value from which the initial state is calculated. It can be a value of any type. How the initial state is calculated from it depends on the next `init` argument.
-* **optional** `init`: The initializer function that should return the initial state. If it's not specified, the initial state is set to `initialArg`. Otherwise, the initial state is set to the result of calling `init(initialArg)`.
+* `reducer`: State'in nasıl güncelleneceğini belirleyen reducer fonksiyonudur. Saf olmalıdır, state'i ve işlemi argüman olarak almalıdır ve bir sonraki state'i döndürmelidir. State ve işlem herhangi bir tür olabilir.
```

Comment:
```suggestion
* `reducer`: State'in nasıl güncelleneceğini belirleyen reducer fonksiyonudur. Saf (pure) olmalı, state'i ve aksiyonu argüman olarak almalı ve bir sonraki state'i döndürmelidir. State ve aksiyon herhangi bir tür olabilir.
```

Reply:
action'in aksiyon olarak cevrilmesini tavsiye eden bir kilavuz var mi? Tum action kelimelerini islem olarak cevirdim.

Reply:
Yani bence ikisi de olabilir. `action.type` -> `islem tipi`. Garip gelmedi gozume. Kalabilir `islem` bence. Sadece reducerdaki `action` cok yaygin olarak issuelarda ve sorularda gectigi icin, bence parantez icinde `(action)` olarak belirtebiliriz. Bunu her yerde yapmamiza gerek yok. Action muhabbetinin ilk gectigi ve aciklandigi yer neresiylse orada belirtsek yeterli.